config/secrets.yml.example

```diff
@@ -0,0 +1,21 @@
+development:
```

Comment:
please add this line from highlights api to the event api dockerfile https://github.com/openstax/highlights-api/blob/master/Dockerfile#L11 to create the secrets.yml